src/compas_fab/ghpython/components/Cf_ConstraintsFromConfiguration/code.py

```diff
@@ -0,0 +1,35 @@
+"""
+Create joint constraints for each of the robot's configurable joints based on a given target configuration.
+
+COMPAS FAB v0.25.0
+"""
+import math
+
+from ghpythonlib.componentbase import executingcomponent as component
+
+
+class ConstraintsFromTargetConfiguration(component):
+
+    DEFAULT_TOLERANCE_METERS = .001
+    DEFAULT_TOLERANCE_RADIANS = math.radians(1)
+
+    def RunScript(self, robot, target_configuration, tolerance_above, tolerance_below, group_name):
```

Comment:
```suggestion
    def RunScript(self, robot, target_configuration, tolerance_above, tolerance_below, group):
```
I would rename this to `group` to match all other components. I guess this works because GH matches the input arguments on the metadata to arguments of this function purely based on position.